files-csharp/CopyAllContentInDirectory/CopyAllContentInDirectoryTests/UnitTest1.cs

```diff
@@ -0,0 +1,74 @@
+using CopyAllContentInDirectory;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Linq;
+using System.IO.Abstractions;
+using Moq;
+using System.IO.Abstractions.TestingHelpers;
+using System.Runtime.InteropServices;
+
+namespace CopyAllContentInDirectoryTests
+{
+    [TestClass]
+    public class UnitTest1
+    {
+        [TestMethod]
+        public void WhenRunningMain_ThenDirectoryDeepCopyOccurs()
```

Comment:
These should be two separate tests. One for Windows and one for Linux.

Reply:
Done!